cmd/minikube/cmd/start.go

```diff
@@ -171,32 +173,43 @@ func init() {
 }
 
 // initClusterUpFlags creates the CLI flags which needs to be passed on to 'oc cluster up'
 func initClusterUpFlags() {
 	//clusterUpFlagSet.StringVar(&clusterUpConfig.Image, "image", "openshift/origin", "Specify the images to use for OpenShift")
-	clusterUpFlagSet.Bool(skipRegistryCheck, false, "Skip Docker daemon registry check")
-	clusterUpFlagSet.String(publicHostname, "", "Public hostname for OpenShift cluster")
-	clusterUpFlagSet.String(routingSuffix, "", "Default suffix for server routes")
-	clusterUpFlagSet.Bool(useExistingConfig, false, "Use existing configuration if present")
-	clusterUpFlagSet.String(hostConfigDir, dockerhost.DefaultConfigDir, "Directory on Docker host for OpenShift configuration")
-	clusterUpFlagSet.String(hostVolumesDir, dockerhost.DefaultVolumesDir, "Directory on Docker host for OpenShift volumes")
-	clusterUpFlagSet.String(hostDataDir, "", "Directory on Docker host for OpenShift data. If not specified, etcd data will not be persisted on the host.")
-	clusterUpFlagSet.Bool(forwardPorts, false, "Use Docker port-forwarding to communicate with origin container. Requires 'socat' locally.")
-	clusterUpFlagSet.Int(serverLogLevel, 0, "Log level for OpenShift server")
-	clusterUpFlagSet.StringSliceVarP(&openShiftEnv, openshiftEnv, "e", []string{}, "Specify key value pairs of environment variables to set on OpenShift container")
+	clusterUpFlagSet.Bool(skipRegistryCheck, false, "Skip the Docker daemon registry check.")
+	clusterUpFlagSet.String(publicHostname, "", "Public host name of the OpenShift cluster.")
+	clusterUpFlagSet.String(routingSuffix, "", "Default suffix for the server routes.")
+	clusterUpFlagSet.Bool(useExistingConfig, false, "Use existing configuration if available.")
+	clusterUpFlagSet.String(hostConfigDir, dockerhost.DefaultConfigDir, "Location of the OpenShift configuration files on the host.")
+	clusterUpFlagSet.String(hostVolumesDir, dockerhost.DefaultVolumesDir, "Location of the OpenShift volumes on the host.")
+	clusterUpFlagSet.String(hostDataDir, "", "Location of the OpenShift data on the host. If not specified, etcd data will not be persisted on the host.")
+	//NEEDINFO: what does etcd data mean? persistent?
+	clusterUpFlagSet.Bool(forwardPorts, false, "Use Docker port forwarding to communicate with origin container. Requires 'socat' locally.")
+	//NEEDINFO: origin or original? origin might not be the right word here. what is socat?
+	clusterUpFlagSet.Int(serverLogLevel, 0, "Set the log level for the OpenShift server.")
+	//NEEDINFO: do we need to default and possible values here? what is this used for, debugging openshift? the app? minishift?
+	clusterUpFlagSet.StringSliceVarP(&openShiftEnv, openshiftEnv, "e", []string{}, "Specify key-value pairs of environment variables to pass to the OpenShift container.")
+	//NEEDINFO: This is the first time I see OpenShift container, isn't this the same as Minishift VM? or app? and what's the difference between this and the Docker daemon env vars?
```

Comment:
> OpenShift container, isn't this the same as Minishift VM?

Absolutely not. The VM is the thing which runs a Docker daemon. On this Docker daemon we are running a containerized version of OpenShift. It is this container to which these settings are relevant.